roles/bind-rhscl-postgresql-apb-openshift/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# only use lowercase letters because postgres does not allow upper case letter,
+# or a number at the start of the username
+postgresql_new_user: "{{ lookup('password', '/tmp/new_username length=15 chars=ascii_lowercase') }}"
```

Comment:
Lookup seems off, do you mean 'user'?

Reply:
I used the ``password`` feature just because it's an easy way to create a random string, even though I intended to use that string as the username.

Reply:
I learned about the `password` feature, after making this comment :sunglasses:.